packages/selinux-policy/fs.cil

```diff
@@ -97,7 +97,8 @@
 (filecon "/run/.*" any ())
 
 ; Label external filesystem mounts.
-(filecon "/mnt" any external)
+(filecon "/mnt" any local)
 (filecon "/mnt/.*" any ())
-(filecon "/media" any external)
+(filecon "/media" any local)
+(filecon "/media/cdrom" any local)
```

Comment:
What's special about cdrom?

Reply:
We create it in the `filesystem` package, and if we don't label it then it can end up with an unexpected SELinux label if we build the image on a host with SELinux enabled.

That happens since we build inside a container, and files end up with container labels in the absence of an override. Our policy has type aliases for the Fedora object types, so they're valid at runtime.

This affected the images I build, and I found it while writing a test to assert that none of the container object labels were present outside `/local`.

Every other file and directory is already covered by a labeling rule.

Reply:
I'm curious whether changing `(filecon "/media/.*" any ())` to `local` instead of `()` would prevent issues with any devices like cdrom that we may add, without having to think about this bit of policy in the future.  I'm not sure what other effects that might have, though, so I'm happy with whatever you like.